.travis.yml

```diff
@@ -21,7 +21,7 @@ matrix:
       env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=1 OMP_NUM_THREADS=2
     - os: linux
       python: "3.6"
-      env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=0 DEVITO_BACKEND=yask
+      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=yask
```

Comment:
Should we update the above `gcc-5` builder to `gcc-7` too?

Reply:
so the only reason I switched to 7 is that I was worried that Travis, or any future CI platform we'll use, would give us a skylake (or newer) architecture; the YASK backend runs some tests with vectorisation, and the architecture autodetection mechanism, based on gcc, won't work if gcc is too old and the architecture is too new. Look at ``infer_cpu`` for more info

Now, to answer your question: I don't think it's too bad to test with a number of gcc versions. It's true it's not consistent, but still it gives us wider coverage. Debatable, though